components/fxa-client/ios/FxAClient/FxAccountManager.swift

```diff
@@ -520,6 +520,15 @@ open class FxAccountManager {
                     )
                 }
             }
+            case .fetchProfile: do {
+                FxALog.info("Fetching profile...")
+                do {
+                    profile = try requireAccount().getProfile()
+                } catch {
+                    return Event.failedToFetchProfile
+                }
+                return Event.fetchedProfile
```

Comment:
There's no transition in `FxAccountState` for this event. I guess we could add a `case .failedToFetchProfile: return .authenticatedNoProfile` OR stay in the current state (after all, we do have a profile). What do you think?